src/css/components/contact.css

```diff
@@ -24,7 +24,7 @@
 .contact .contact__headline-text {
   font-size: var(--text-lg);
   line-height: 28px;
-  color: var(--color-primary-B50);
+  color: var(--color-neutrals);
 }
 
 .contact .contact__form {
```

Comment:
- [ ] Please work on the color contrast of the `button Get In Touch`  I suggest that you re-adust the border color when Hover. 
- See Details
![c2](https://user-images.githubusercontent.com/63972670/166482814-a2e3f53d-49a6-47e0-a0a1-7f08843bd216.PNG)

- [ ] More Details:
![c](https://user-images.githubusercontent.com/63972670/166483123-0a05b178-1008-4351-b9ac-f60290bd9d01.PNG)